.github/workflows/release.yml

```diff
@@ -18,48 +31,328 @@ jobs:
       fail-fast: false
       matrix:
         go: [ "1.16.10" ]
```

Comment:
What about removing this matrix.go from each of the jobs and make them consume the go version from the "global" env GO_VERSION?

Reply:
I chose the matrix as it allows for running with multiple go versions. Once v1.17 is consumed by flannel itself we could add it in the array and the automation would run for both 1.16.x and 1.17.x. I figured this would make automated smoke tests and general tests much easier to diagnose going forward

Reply:
ok. I was more wondering if we could control the go versions from a "global" variable on top of the file which would simplify our lives when, e.g. adding more versions. Now, as we have several jobs, we will need to change it in several places in the file but it's fine